connectivity/drivers/lora/TARGET_STM32WL/mbed_lib.json

```diff
@@ -47,6 +47,10 @@
         },
         "debug_tx": {
             "help": "GPIO pin for TX debug"
+        },
+        "debug_led_off": {
```

Comment:
I would like to avoid name with "led", as debug pins are only GPIO

Reply:
It was @0xc0170 proposal to add `led` to my first proposal. So do I need to revert back to my original name `debug_invert`? could you please agree on a name so I don't need to change back again. Thanks

Reply:
It could be my misunderstanding this is always connected to LED but if it can be be GPIO (output in this case), should debug_output_off be a better name? I find `debug_off` to be more related to turn off entire debug unit than just one pin. Or `debug_pin_off` could be as well one alternative.

Reply:
@jeromecoutant you know better so suggest one to be used.

Reply:
Just to not forgot this one, what name should we use on this? @jeromecoutant

Reply:
A name with "debug" and not "led" :-)

Reply:
ok, done, not sure why ci not started since...